src/parser/if-then-else.js

```diff
@@ -26,7 +26,12 @@ const ifThenElse = (ctx: Context) => {
 
     ctx.expect(["}"]);
 
-    if (ctx.eat(["else"])) {
+    while (ctx.eat(["else"])) {
```

Comment:
Looks like we are not handling the case where an engineer is omitting the `{}` brackets and using single statements.

```js
if (x == 0) 
  return 1;
else if (x == 1)
  return 2;
else 
  return -1;
```

We should do that before we enable `else if` statements